torch/nn/modules/linear.py

```diff
@@ -115,4 +116,109 @@ def __repr__(self):
             + ', in2_features=' + str(self.in2_features) \
             + ', out_features=' + str(self.out_features) + ')'
 
+
+class NoisyLinear(Module):
+    """Applies a noisy linear transformation to the incoming data.
+    During training:
+        :math:`y = (mu_w + sigma_w \cdot epsilon_w)x
```

Comment:
[Sphinx maths docs](http://www.sphinx-doc.org/en/stable/ext/math.html). So yes Aly is right.